games-puzzle/gemdropx/gemdropx-0.9-r3.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop toolchain-funcs
+
+DESCRIPTION="Puzzle game where it's your job to clear the screen of gems"
+HOMEPAGE="http://www.newbreedsoftware.com/gemdropx/"
+SRC_URI="ftp://ftp.tuxpaint.org/unix/x/gemdropx/src/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	media-libs/libsdl[joystick,sound,video]
+	media-libs/sdl-mixer[mod]
+	!!sci-biology/unafold"
```

Comment:
Two things here.

1.
A weak-blocker (single `!`) is enough to prevent file-collisions, it tells portage it can ignore collisions and unmerge unafold afterwards.

A strong `!!` tells portage it /must/ unmerge first and this can sometime create unsolvable situations needing manual user interaction (users often end up asking for support not knowing what to do), it's best avoided where possible. Primary case where you'd use that is if the presence of the package would cause a build failure, but here we have no direct relation to unafold.

https://devmanual.gentoo.org/general-concepts/dependencies/#blockers

2.
This is supposed to be in cencentration, not gemdropx :)

Reply:
Oh and don't forget to revbump concentration when adding that.

Reply:
I didn't immediately spot that (essentially the only) example in the devmanual, but if it's not there, would you mind doing a PR to add it?

Reply:
Spot what? The devmanual gives the weak blocker as its first example and mentions file collisions, so it's probably enough.

Revbump-wise, revbump is only needed blocker is one-sided (rather than added to both) -- but generally it's a RDEPEND change nonetheless.

Reply:
Explicitly stating that a strong blocker is pretty much only for where something else being installed calls a failure to compile?

Reply:
Oh I see, well it does say `if strictly necessary` -- that implies you're supposed to have a reason that goes beyond the aforementioned file collisions.

Reply:
yep, just wanted to add a concrete example/essentially the only reason to actually do this given the misunderstandings

Reply:
@ionenwks 
fixed and updated my PR. Thanks again for the explanation :)